templates/singular-category.html

```diff
@@ -0,0 +1,9 @@
+{% if post.category.name == 'Articles' %}Article{% endif %}
+
+{% if post.category.name == 'Case studies' %}Case study{% endif %}
+
+{% if post.category.name == 'News' %}News{% endif %}
+
+{% if post.category.name == 'Webinars' %}Webinar{% endif %}
+
+{% if post.category.name == 'White papers' %}White paper{% endif %}
```

Comment:
We should have a catch if the category is not known. For example, if we add a new one it should just output `post.category.name`.